File: Questão 1/Q1A.js
function Verificar(num){
	let n = num.toString()
	let tt = true
	
	for(let i = 0; i < n.length;i++){
		if(n[i] == '0' || n[i] == '1'){
			
		} else {
			tt = false
			break
		}
	}
	if(tt) return true
	else return false
}

let numero = 9

while(true){
	if(Verificar(numero)){
		console.log(`Menor numero encotrado: ${numero}`)
		break
	}
	numero += 9
}

File: Questão 1/Q1A.cs
using System;

class Q1A{
	public static bool Verificar(int num){
		string n = Convert.ToString(num);
		bool tt = true;
		
		for(int i = 0; i < n.Length;i++){
			if(n[i] == '0' || n[i] == '1'){
				
			} else {
				tt = false;
				break;
			}
		}
		
		if(tt) return true;
		else return false;
	}
	
	public static void Main(){
		int i = 9;
		while(true){
			if(Verificar(i)){
				Console.WriteLine("Menor valor encontrado: " + i);
				break;
			}
			i += 9;
		}
		Console.ReadKey();
	}
}